ignite/chainconfig/common/config.go

```diff
@@ -0,0 +1,162 @@
+package common
+
+type Version int
+
+var (
+	// LatestVersion defines the latest version of the config.
+	LatestVersion Version = 1
```

Comment:
This is not something can be considered "common". Can we move this to a new file called `chainconfig/config.go`? pkg?